en/10_Contributing/02_Documentation.md

```diff
@@ -93,30 +93,53 @@ yarn lint --fix
 
 ## Writing style
 
-- Write in second person form, addressing the reader. For example "First you'll install a webserver" is good style.
```

Comment:
I'm personally not a fan of the "Write in second person form" because IMO it  just adds unnecessary fluff. I much prefer dry technical documentation that simply tells you what to do "Install a webserver".

Reply:
Also prefer direct and instructional approach rather than a story.

Reply:
I think that one was more intended to prefer "you can configure this by..." instead of "this can be configured by..." which is just another way of saying "use an active voice".